modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -4,30 +4,30 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-installing-aws-load-balancer-operator_{context}"]
-= Installing the AWS Load Balancer Operator
+= Installing AWS Load Balancer Operator by using the web console
```

Comment:
I would leave the _the_.

Reply:
Especially where you added a _the_ in the [following assembly](https://github.com/openshift/openshift-docs/pull/73583/files#diff-04550ce247b3f955736f564779b31dd454d1295da1e2fb50d07fe37817511ad1R3).

Reply:
Fixed, thank you